Formula/corectl.rb

```diff
@@ -13,26 +20,51 @@ class Corectl < Formula
 
   depends_on "go" => :build
   depends_on "godep" => :build
+  depends_on "ocaml" => :build
+  depends_on "opam" => :build
   depends_on :macos => :yosemite
 
   def install
     ENV["GOPATH"] = buildpath
 
+    opamroot = buildpath/"opamroot"
+    opamroot.mkpath
+    ENV["OPAMROOT"] = opamroot
+    ENV["OPAMYES"] = "1"
+
     path = buildpath/"src/github.com/TheNewNormal/#{name}"
     path.install Dir["*"]
 
     args = []
     args << "VERSION=#{version}" if build.stable?
 
     cd path do
-      system "make", "corectl", *args
-      system "make", "documentation/man"
-      bin.install "corectl"
+      system "make", "default", *args
+
+      system "opam", "init", "--no-setup"
+      system "opam", "pin", "add", "qcow-format",
+        "https://github.com/mirage/ocaml-qcow.git#master"
+      system "opam", "install", "uri", "qcow-format", "ocamlfind"
+      system "make", "hyperkit"
+
+      bin.install "bin/corectl"
+      bin.install "bin/corectld"
+      bin.install "bin/corectld.runner"
+
       man1.install Dir["documentation/man/*.1"]
-      share.install "cloud-init", "profiles"
+      pkgshare.install "examples/cloud-init", "examples/profiles"
```

Comment:
Wouldn't be simpler to `pkgshare.install "examples"`? Are there ones you're deliberately not installing for some reason?